prow/plugins.yaml

```diff
@@ -13,22 +13,28 @@ triggers:
   trusted_org: kubernetes
   join_org_url: "https://git.k8s.io/community/community-membership.md#member"
   only_org_members: true
+  elide_skipped_contexts: true
```

Comment:
Is this not the default? Can we make it the default?

Reply:
I [mentioned](https://github.com/kubernetes/test-infra/issues/12047#issuecomment-532903026) this in issue. It has been more than 6 months so we're probably good to switch the default (and maybe remove the flag entirely).
Would you rather we just do that instead of adjusting the config here first? I think doing this config change first is a bit safer and easier to revert if we find "Skipped" statuses are important for some case.

Reply:
I [mentioned](https://github.com/kubernetes/test-infra/issues/12047#issuecomment-532903026) this in issue. It has been more than 6 months so we're probably good to switch the default (and maybe remove the flag entirely).
Would you rather we just do that instead of adjusting the config here first? I think doing this config change first is a bit safer and easier to revert if we find "Skipped" statuses are important for some case.

Reply:
May be easier to start by flipping the flag here -- we don't have evidence the system works correctly with it the other way -- but I think we should also flip the default very soon.